packages/o/obs-studio/package.yml

```diff
@@ -28,23 +31,35 @@ builddeps  :
     - pkgconfig(luajit)
     - pkgconfig(nlohmann_json)
     - pkgconfig(python3)
+    - pkgconfig(rnnoise)
     - pkgconfig(speexdsp)
     - pkgconfig(srt)
+    - pkgconfig(vpl)
     - pkgconfig(wayland-client)
     - pkgconfig(x264)
     - pkgconfig(xcomposite)
     - pkgconfig(xinerama)
     - pkgconfig(xkbcommon-x11)
+    - asio
     - cef-minimal-devel
+    - fdk-aac-devel
+    - libdatachannel-devel
```

Comment:
We can exclude this dep & package if we want to turn off the WebRTC feature `ENABLE_WEBRTC`